go/base_client.go

```diff
@@ -76,7 +76,7 @@ func (client *baseClient) getMessageHandler() *MessageHandler {
 func (client *baseClient) GetQueue() (*PubSubMessageQueue, error) {
 	// MessageHandler is only configured when a subscription is defined
 	if client.getMessageHandler() == nil {
-		return nil, &errors.RequestError{Msg: "No subscriptions configured for this client"}
+		return nil, errors.New("No subscriptions configured for this client")
```

Comment:
error strings should start with lower case [^1].

[^1]: https://go.dev/wiki/CodeReviewComments#error-strings

Reply:
here and elsewhere in the code.

Reply:
:) I see the linter is complaining about this as well.

Reply:
done.